### ISSUE TITLE
add other weather station or your own station 

### ISSUE THREAD

**Issue by elRadix** [Ernst79/degree-days#11]
in Belgium we use other stations... or give the option select your your own station in home assistant ie darksky etc

**Comment by elRadix** [comment]
any progress on that? I'd like to select my own weather entity

**Comment by Ernst79** [comment]
No, sorry. Im afraid i wont have time to add this. But feel free to make a PR if you want to add functionality yourself.

**Comment by nelbs** [comment]
I don't think that is going to work because the integration needs historical weather data of the past years too to make a good prognose.
